Source/Plugins/Core/com.equella.core/js/tsrc/components/DateRangeSelector.tsx

```diff
@@ -248,8 +249,8 @@ export const DateRangeSelector = ({
         value={dateRangeToDateOptionConverter(stateDateRange)}
         id="date_range_selector"
         labelId="date_range_selector_label"
-        onChange={(event) =>
-          handleQuickDateOptionChange(event.target.value as string)
+        onChange={(event: any) =>
```

Comment:
event should automatically have a type here:

```suggestion
        onChange={(event) =>
```

Reply:
Putting any as the type was to get around event.target.value now being an Unknown type. 
ESLint complains aboutthe string assertion 'This assertion is unnecessary since it does not change the type of the expression'

But i'm open to other ways to get around this. Since using `any` feels like a bit of a hack

Reply:
try switching `target` to `currentTarget`.
`target` bubbles events up, if there were another input inside the input it would catch that, so it's type is dynamic and unknown.
`currentTarget` refers to the element which the listener is attached to, so it can be statically typed.

Reply:
That seems to work nicely, thank you for your suggestion  @ChristianMurphy :+1: :+1: